libtenzir/builtins/formats/zeek_tsv.cpp

```diff
@@ -648,12 +647,7 @@ auto parser_impl(generator<std::optional<std::string_view>> lines,
           }
           return false;
         });
-      if (target_schema != ctrl.schemas().end()
-          and can_cast(document.builder->schema(), *target_schema)) {
-        document.target_schema = *target_schema;
-      } else {
-        document.target_schema = document.builder->schema();
-      }
+      document.target_schema = *target_schema;
```

Comment:
Possibly related to this change, which I do not understand?

Reply:
Looks like I pushed some in-between state, sorry 🙈 I pushed everything now. Mind taking another look?

Reply:
This works now, thanks :+1: